### PR TITLE
[WIP] KEP: Switch kubeadm to generate ECDSA keys

### PR DESCRIPTION
**Pull request by rojkov**

### CODE REVIEW

**Comment by neolit123** [comment]
thanks @rojkov will attempt to have a closer look later today or tomorrow.
deadline for KEPs, as far as i know is 30th of April.

File: keps/sig-cluster-lifecycle/kubeadm/20190423-switch-to-ecdsa-keys.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: Switching kubeadm to generate ECDSA keys
```
Comment (timothysc) [inline keps/sig-cluster-lifecycle/kubeadm/20190423-switch-to-ecdsa-keys.md]: Why is this a KEP?  You can just open an issue for this in the kubeadm repo...

https://github.com/kubernetes/kubeadm

**Comment by timothysc** [comment]
/hold 

IMO we should close this KEP, it's not a feature or major enhancement, we can just treat it like an ordinary issue.

**Comment by rojkov** [comment]
Sure, if there's a consensus I'm fine with closing it. The only reason for the KEP is that it can span several releases.

**Comment by fabriziopandini** [comment]
ok for me in moving this to an issue

**Pull request closed without merging**